cmd/webhook-manager/app/options/options.go

```diff
@@ -83,6 +84,7 @@ func (c *Config) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&c.SchedulerName, "scheduler-name", defaultSchedulerName, "Volcano will handle pods whose .spec.SchedulerName is same as scheduler-name")
 
 	fs.StringVar(&c.ConfigPath, "admission-conf", "", "The configmap file of this webhook")
+	fs.StringVar(&c.IgnoredNamespaces, "ignored-namespaces", "volcano-system,kube-system", "Comma-separated list of namespaces to be ignored by admission webhooks")
```

Comment:
How abount use `fs.StringSliceVar()`.
Feels unreliable to split strings manually
And define a const named `defaultIgnoredNamespaces` in https://github.com/volcano-sh/volcano/blob/718b91321f87d9f4f3fb5dcce8fee4be16049149/cmd/webhook-manager/app/options/options.go#L28

Reply:
I think we will need to choose one or the other:
- We can switch to `fs.StringSliceVar()` and inline the default (golang doesn't allow const string slices)
- _Or_, we can keep the flag as a string and manually split the string, following the pattern used by the `EnabledAdmission` flag.

I'll assume it's preferred to follow the existing patterns in the codebase and will set up a string const with trimming and slicing after. I don't believe this poses a significant risk since input to CLI flag is tightly controlled by the cluster admin.